app/views/memberships/_table.html.erb

```diff
@@ -51,11 +51,11 @@
 
         <div class='govuk-grid-column-one-third govuk-list govuk-!-padding-right-4 govuk-!-margin-top-0 text-right'>
           <% if current_user.can_manage_team?(current_organisation) && member.id != current_user.id %>
-            <%= link_to "Edit permissions", edit_membership_path(member.membership_for(current_organisation)), class: "govuk-link govuk-link--no-visited-state" %>
+            <%= link_to "Edit permissions", edit_membership_path(member.membership_for(current_organisation)), class: "membership-list govuk-link--no-visited-state" %>
```

Comment:
I think I would agree with Robin's previous comment here: style the parent `ul` element and then you can just do the styling within the scope of that (as opposed to tagging both elements with it). And then you can group the CSS styling mentioned above.

Reply:
Good plan - I've now done this.